### PR TITLE
Removed redundant code

### PR DESCRIPTION
**Pull request by suhasasumukh**
Used `map` and `join` methods for better code clarity

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -18,6 +18,28 @@ const dbConfig = {
...
+    if (!cronUnit) {
+        throw new Error('Invalid time format');
+    }
+
+    const cronTime = unit === 's' ? `*/${value} * * * *` : unit === 'd' ? `0 0 */${value} * *` : `0 */${value} * * *`;
+    const sqlInterval = `INTERVAL ${value} ${cronUnit}`;
+    
+    return [cronTime, sqlInterval];
+};
+
```
Comment (RevTpark) [inline index.ts:42]: This extraction and implementation can be kept in the PR, all the other stuff is unnecessarily extracted for a such a simple bot.

**Review by RevTpark — CHANGES_REQUESTED** [review]

File: index.ts (hunk 1 above)
Comment (suhasasumukh) [reply index.ts]: I have the `getCronAndSqlInterval` function implemented in the code.

Comment (RevTpark) [reply index.ts]: Yea just keep that code, i dont see the point of all the other extractions

**Comment by RevTpark** [comment]
Closing since
- Unnecessary commits
- Raise a PR with just the `getCronAndSqlInterval` and its call in the code.

**Pull request closed without merging**